app/assets/javascripts/oxalis/model/reducers/reducer_helpers.js

```diff
@@ -32,6 +34,16 @@ export function convertFrontendBoundingBoxToServer(
     .getOrElse(null);
 }
 
+export function convertBoundariesToBoundingBox(boundary: Boundary): BoundingBoxObject {
+  const [width, height, depth] = V3.sub(boundary.upperBoundary, boundary.lowerBoundary);
+  return {
+    width,
+    height,
+    depth,
+    topLeft: boundary.lowerBoundary,
+  };
+}
+
 export function convertPointToVecInBoundingBox(boundingBox: ServerBoundingBox): BoundingBoxObject {
```

Comment:
Was this function unused now?
But we can also check that when we refactor the bounding box/upper-lower boundary mess, maybe add a comment.

Reply:
Yes, it's unused. I'd leave it as is, though, since the cost of recreating the function (in case it's needed again) is higher than having the few additional unused lines in the code base. Getting rid of some bounding box types will be more rewarding :)